src/native/libs/System.Native/pal_mount.c

```diff
@@ -132,48 +132,43 @@ int32_t SystemNative_GetSpaceInfoForMountPoint(const char* name, MountPointInfor
 }
 
 int32_t
-SystemNative_GetFormatInfoForMountPoint(const char* name, char* formatNameBuffer, int32_t bufferLength, int64_t* formatType)
+SystemNative_GetFileSystemTypeNameForMountPoint(const char* name, char* formatNameBuffer, int32_t bufferLength)
 {
-    assert((formatNameBuffer != NULL) && (formatType != NULL));
+    assert(formatNameBuffer != NULL);
```

Comment:
I've removed the use of `stats.f_type` here. The managed side expects the Linux values to be returned. With the changes in this PR, we're no longer calling this on Linux, and other OSes are not expected to return the same values as Linux.